assets/app/scripts/controllers/create/createFromImage.js

```diff
@@ -137,7 +142,14 @@ angular.module("openshiftConsole")
                         message: "Cannot create " + objectName + ". ",
                         details: failure.data.message
                       });
-                      hasErrors = true;
+                    }
+                  );
+                  result.success.forEach(
+                    function(success) {
+                      alerts.push({
+                        type: "success",
+                        message: "Created " + success.kind.toLowerCase() + " \"" + success.metadata.name + "\" successfully. "
```

Comment:
We have said in errors they should be human focused - i.e. "Deployment
config"

On Sep 23, 2015, at 7:52 PM, Jessica Forrester <notifications@github.com>
wrote:

In assets/app/scripts/controllers/create/createFromImage.js
<https://github.com/openshift/origin/pull/4728#discussion_r40272685>:

> @@ -137,7 +142,14 @@ angular.module("openshiftConsole")
>                          message: "Cannot create " + objectName + ". ",
>                          details: failure.data.message
>                        });
> -                      hasErrors = true;
> +                    }
> +                  );
> +                  result.success.forEach(
> +                    function(success) {
> +                      alerts.push({
> +                        type: "success",
> +                        message: "Created " + success.kind.toLowerCase() + " \"" + success.metadata.name + "\" successfully. "

The message is more readable if kind is "deployment config" or even
"deployment configuration" vs "deploymentconfig", same with any of the
multi-word kinds. The existing failures are even different, they seem to
show it in camel case as "deploymentConfig". So consistency one way or
another would be nice :)

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4728/files#r40272685>.